app/serializers/provider_serializer.rb

```diff
@@ -35,6 +35,16 @@ class ProviderSerializer < ActiveModel::Serializer
              :contact_name, :email, :telephone, :recruitment_cycle, :utt_application_alerts,
              :type_of_gt12
 
+             # application alert reciepient has not been added into the enum in the contact model
+             # as it's does not share the name and email attribute. It is simply a contact email
+             # address and is more suited to sit in the ucas  preferences model. However, in the docs
+             # it is exposed in the contacts so has been added directly into the contacts attribute
+             # see lines 109-120 for logic.
```

Comment:
drop lines number just refer to those methods...
spacing of comment blocks you referring to `attributes ` or `attribute :contacts do`

ie line numbers will jump around

typo `reciepient` 
grammer `it's` does not share
double space ucas`  `preferences model